Sources/_SwiftSyntaxTestSupport/LocationMarkers.swift

```diff
@@ -38,11 +38,6 @@ fileprivate func nextMarkedRange(text: String, from: String.Index) -> Marker? {
   let markerRange = start ..< end
   let name = text[start ..< end]
 
-  // Expand to the whole line if the line only contains the marker
-  let lineRange = text.lineRange(for: start..<start)
-  if text[lineRange].trimmingCharacters(in: .whitespacesAndNewlines) == text[markerRange] {
-    return Marker(name: name, range: lineRange)
-  }
   return Marker(name: name, range: markerRange)
```

Comment:
FWIW after your explanation I wasn't expecting you to remove this. But we can always add it back if we need it :)

Reply:
Thinking about it again, I couldn’t really find a good use case, so I though we might just simplify things.